packages/server/src/adapters/aws-lambda.ts

```diff
@@ -0,0 +1,102 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { URLSearchParams } from 'url';
+import { resolveHTTPResponse } from '..';
+import {
+  HTTPBaseHandlerOptions,
+  HTTPHeaders,
+  HTTPRequest,
+} from '../http/internals/types';
+import { AnyRouter, inferRouterContext } from '../router';
+import { NodeHTTPCreateContextFn } from './node-http';
```

Comment:
As @mmkal mentioned - this needs to go and be implemented with a lambda-specific thing